src/Section1/Part1/Lesson1.md

```diff
@@ -2,7 +2,7 @@
 
 Let’s start from sentences in the shape of “A is B”. Japanese has two basic "is" words. The grammar term for such words is "copula". Copulas have a dedicated term because they're special verb-like things.
 
-The two copulas in Japanese are だ and です. です is more polite than だ. Despite both being copulas, their usage can be flexible and sometimes optional: you can omit them entirely while still implying an “A is B” type of relationship in your sentences. Especially だ is usually omitted at the end of conversational sentences in modern Japanese.
+The two copulas in Japanese are だ and です, and です is more polite than だ. Despite both being copulas, their usage can be flexible and sometimes optional; you can omit them entirely while still implying an “A is B” type of relationship in your sentences. Especially だ is usually omitted at the end of conversational sentences in modern Japanese.
```

Comment:
I prefer to keep the sentences separated with a period rather than add a comma + "and".  Just personal preference but also I feel like it adds less burden on the mind of the reader if they don't have to parse longer sentences and instead focus on short and concise ones instead.

As for the ": -> ;" change, that looks good to me. Thanks